draft-ietf-sipcore-sip-push.xml

```diff
@@ -680,7 +695,7 @@ align="center"><artwork>
 
     <section anchor="section.sec-webpush" title="pn-provider, pn-param and pn-prid URI Parameters for RFC 8030 (Generic Event Delivery Using HTTP Push)">
       <t>
-        When Firebase Cloud Messaging (FCM) is used, the PNS related URI parameters are set as described below.
+        When Generic Event Delivery Using HTTP Push is used, the PNS related URI parameters are set as described below.
```

Comment:
I agree with not defining this as FCM but I'd prefer to leave in some reference to FCM to help implementors easily find what they're looking for. Suggest changing to:
"When Generic Event Delivery Using HTTP Push is used (e.g. Firebase Cloud Messaging)"

Appreciate others may feel differently on this and I don't feel too strongly about it.

Reply:
Note that there is a separate section about FCM. This section is about the generic push mechanism - the FCM text was a copy/paste error.

Reply:
Sorry missed that there was another FCM section!